.remarkrc

```diff
@@ -1,12 +1,14 @@
 {
   "plugins": [
-    "preset-lint-markdown-style-guide",
+    ["preset-lint-markdown-style-guide", true],
+    ["preset-lint-recommended", true],
     ["lint-no-duplicate-headings", false],
     ["lint-list-item-indent", "tab-size"],
     ["lint-emphasis-marker", "consistent"],
     ["lint-maximum-line-length", false],
     ["lint-maximum-heading-length", false],
     ["lint-no-shortcut-reference-link", false],
-    ["lint-no-trailing-spaces"]
+    ["lint-no-trailing-spaces"],
+    ["lint-no-undefined-references", false]
```

Comment:
this rule needed to be added to make remark stop complaining about macros such as:  `{{ MACROS___make_filename_template(datatypes=["eeg"], suffixes=["electrodes"]) }}`